authorizer-app-backend/src/main/java/org/radarbase/authorizer/api/Project.kt

```diff
@@ -39,5 +39,5 @@ fun MPUser.toUser() = User(
         id = id,
         projectId = checkNotNull(projectId) { "User must have a project ID" },
         externalId = externalId,
-        status = status,
+        status = status
```

Comment:
The comma was intentional. You can set this as default behaviour in IntelliJ IDEA by going to Editor -> Code Style -> Kotlin -> Other tab -> enable trailing comma.